tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -136,27 +136,9 @@ Filtering:
   ExpFilter0:
     Alpha: 36.0
     HalfPower: 210
-    DisableForDebugging: false
+    DisableForDebugging: true
```

Comment:
This says "for debugging". Did you intend to change this?

Reply:
I could change it;  @geoffrey4444 wanted me to use his input file, and this was present in his input file.

Reply:
I no longer recommend filtering for BBH runs, since it doesn't seem necessary in my tests, but I'm unsure whether to remove it altogether, so I prefer disabling in yaml for now